media/js/firefox/hello/index.js

```diff
@@ -127,6 +136,13 @@
                     $('.try-hello').on('click', function(e) {
                         e.preventDefault();
 
+                        // (bug 1115227) pass source to FTU
+                        if (tourSource === 'none') {
+                            Mozilla.UITour.registerPageID('hello-tour_OpenPanel_productPage');
+                        } else {
+                            Mozilla.UITour.registerPageID('hello-tour_OpenPanel_' + tourSource);
```

Comment:
@ckprice - seeing as these are page id's registered with Telemetry, I think we should try and retain tighter control over what is set, as opposed to setting arbitrary values passed via URL parameter. Thoughts?